static/styles/settings.css

```diff
@@ -28,6 +28,15 @@ h3 .fa-question-circle-o {
     left: 5px;
 }
 
+div.settings-section-title a {
+    color: inherit;
+}
+
+div.settings-section-title a i {
+    opacity: 0.6;
+    margin-left: 5px;
+}
```

Comment:
No need to duplicate the css here, we can just use the already existing css for `h4` or `h3` elements in this file itself.

Reply:
The css here refers to the icon for "Add a new code playground" and the question icon was wrapped inside a div element and I could not use the css defined for icons under h3 and so, I created a new css specifically for the icon to change its spacing and opacity.

Reply:
So, you can just change the existing CSS as they finally have same opacity values and adding top value makes the playground icon also looks better.
```diff
--- a/static/styles/settings.css
+++ b/static/styles/settings.css
@@ -8,7 +8,8 @@ label {
 
 label,
 h4,
-h3 {
+h3,
+.settings-section-title {
     a {
         color: inherit;
     }
@@ -24,8 +25,11 @@ h3 {
     }
 }
 
-h3 .fa-question-circle-o {
-    left: 5px;
+h3,
+.settings-section-title {
+    .fa-question-circle-o {
+        left: 5px;
+    }
 }

```

Reply:
Ok got it. I changed the css accordingly and will push the changes.